Sources/Operators/WithLatestFrom.swift

```diff
@@ -8,6 +8,8 @@
 
 import Combine
 
+// swiftlint:disable large_tuple
```

Comment:
Where are we using a large tuple? Might be worth just relaxing that rule

Reply:
Without disabling it, we’re hitting three warnings in the file.
<img width="416" alt="Screen Shot 2020-05-03 at 6 25 06 PM" src="https://user-images.githubusercontent.com/1276296/80927376-7a3c2300-8d6b-11ea-8f87-fc880af68ad6.png">